eth_crowdfund_be/eth_crowdfund_api/src/tests/test_sample.py

```diff
@@ -0,0 +1,5 @@
+def func(x):
+  return x + 1 
+
+def test_answer():
+  assert func(3) == 4
```

Comment:
This is a basic test that should pass without any database or app involvement. to run individually its `pytest <path to file depending on current directory>`  ex: `pytest src/tests/test_sample.py`